modules/roomify/roomify_accommodation_example_content/roomify_accommodation_example_content.module

```diff
@@ -1186,6 +1196,19 @@ function roomify_accommodation_example_content_multi_property_delete($type_delet
   variable_del('roomify_accommodation_example_content_property_id');
   variable_del('roomify_accommodation_example_content_conversation_id');
 
+  // Delete all Homepage nodes.
```

Comment:
let's add to the form where users can choose what content to delete so that they can choose to leave any homepage nodes they might have created themselves.